tests/Avalonia.Controls.UnitTests/MaskedTextBoxTests.cs

```diff
@@ -0,0 +1,952 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Avalonia.Controls.Presenters;
+using Avalonia.Controls.Primitives;
+using Avalonia.Controls.Templates;
+using Avalonia.Data;
+using Avalonia.Input;
+using Avalonia.Input.Platform;
+using Avalonia.Media;
+using Avalonia.Platform;
+using Avalonia.UnitTests;
+using Moq;
+using Xunit;
+
+namespace Avalonia.Controls.UnitTests
+{
+    public class MaskedTextBoxTests
```

Comment:
Some tests in this file are ported from TextBox because MaskedTextBox actually inherits TextBox.